docs/runbooks/maintenance_mode/manage_maintenance.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
Wondering about scripts: should these be mixed in with the runbooks or further up the project?

Reply:
IMO, best to keep these with the runbooks as close to the instructions as possible, for easier access- these tend to be troubleshooting and operational scripts that help remedy a situation. We do have a scripts folder also but these tend to align with the pipeline.

Reply:
Also other teams have adopted a similar pattern.